src/SUMMARY.md

```diff
@@ -1,5 +1,15 @@
 # Summary
 
-- [Onion client](./onion_client.md)
-- [Messenger packets](./messenger_packets.md)
-- [Conference packets](./conference_packets.md)
+[Introduction](intro.md)
+
+- [Cryptography](crypto/crypto.md)
+- [Network](network/network.md)
+- [DHT](dht/dht.md)
+- [TCP Relay](relay/relay.md)
+- [Onion](onion/onion.md)
+    - [Onion client](./onion/client.md)
+- [Transport Protocol](transport/transport.md)
```

Comment:
What will be Transport about?

Reply:
Tox transfer protocol aka netcrypto.

Reply:
Rename to NetCrypto

Reply:
I'm not sure if this is a proper name for a chapter. `Transport Protocol` immediately tells the purpose while `NetCrypto` is opaque.

Reply:
cc @kurnevsky

Reply:
I think it's ok to call it `Transport Protocol` - it's called so in [toktok docs](https://toktok.ltd/spec.html#transport-protocol) as well. Though they have separate [chapter](https://toktok.ltd/spec.html#transport-protocol) where TCP/UDP is called as `Transport Protocol` so it might be slightly confusing.

Reply:
There should be 2 subchapters called Protocol not 1 and not at the top level.

Reply:
Let's add it later.